aiomisc/log/config.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+import sys
+from typing import Union
+
+from .enum import LogFormat
+from .formatter import json_formatter, color_formatter
+from .wrap import wrap_logging_handler
+
+DEFAULT_FORMAT = '%(levelname)s:%(name)s:%(message)s'
+
+
+def create_logging_handler(log_format: LogFormat = LogFormat.color,
```

Comment:
Why we don't allow users to pass their own formatter?
Especially when functions and classes inside `aiomisc.log.formatter` has some degree of customization.

Reply:
This PR just about moving the code and doesn't brings a new features.